.codecov.yaml

```diff
@@ -0,0 +1,6 @@
+ignore:
```

Comment:
May want to use `behavior=new` https://docs.codecov.io/docs/pull-request-comments#behavior so that the Cov always show near the latest of PR comments?

Reply:
that's probably spammier.

Reply:
both behavior keep one message, but `behavior=new` delete the old one while the default behavior updates.

Reply:
Editing doesn't send mail, delete and recreate does.  this is why Prow generates so much email.